src/stories/containers/users/delete-account/delete-account.tsx

```diff
@@ -7,16 +7,33 @@ import AvatarPlaceholder from '../../../components/svg/avatar-placeholder';
 import TextInput from '../../../components/text-input/text-input';
 import { Spacer, Username, UserWrapper } from '../../auth/change-password/change-password';
 import { ButtonWrapper, Container, Wrapper } from '../../auth/login/login';
+import { useRouter } from 'next/router';
+import { useAuthContext } from '../../../../core/context/AuthContext';
+import { USERS_DELETE_FROM_ADMIN } from './delete-account.api';
 
 export default () => {
   const testingPassword = '1234';
```

Comment:
is this still needed?

Reply:
Yes for now, we need until from API create a consult to verify the passoword, later I will make a work around make a login mutation again to verify the password